ios/MOSIPResidentApp/Info.plist

```diff
@@ -48,6 +48,10 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>Allow $(PRODUCT_NAME) to access your location</string>
+    <key>NSLocationWhenInUseUsageDescription</key>
```

Comment:
Take care of formatting here.

Reply:
In vs code, formatting was good. In github, it shows up in this way.

Reply:
Addressed it